api/src/main/java/org/openmrs/module/mirebalais/MirebalaisHospitalActivator.java

```diff
@@ -170,7 +175,21 @@ public void started() {
         log.info("Mirebalais Hospital Module started");
     }
 
-	/**
+    /**
+     * Include custom styling sheets and scripts
+     * @param config
+     */
+    private void includeGlobalResources(Config config) {
+        PageFactory pageFactory = Context.getRegisteredComponents(PageFactory.class).get(0);
+        GlobalResourceIncluder globalResourceIncluder = new GlobalResourceIncluder();
+        String cssResourcePath = "styles/".concat(config.getCountry().name().toLowerCase()).concat(".").concat(Resource.CATEGORY_CSS);
+        String jsResourcePath = "scripts/".concat(config.getCountry().name().toLowerCase()).concat(".").concat(Resource.CATEGORY_JS);
+        globalResourceIncluder.addResource(new Resource(Resource.CATEGORY_CSS, MirebalaisConstants.MIREBALAIS_MODULE_ID, cssResourcePath, -100));
+        globalResourceIncluder.addResource(new Resource(Resource.CATEGORY_JS, MirebalaisConstants.MIREBALAIS_MODULE_ID, jsResourcePath, null));
+        pageFactory.getModelConfigurators().add(globalResourceIncluder);
```

Comment:
Why the -100 and the null for rankings?  I think this is the "priority" of the resource, which I assume determines the order things are loaded.  Maybe we need to load the CSS later to make sure the main CSS is already loaded?  That's fine, but let's add a comment so people know not to change it.

Also, if we don't have any custom JS to install, let's just comment out that line for now...

And finally, it would be better if we only added the css file if we confirmed that a css file already existing for that country?  Actually, you could just create blank css files for each country, with a comment for "add your custom css here"...  that's probably easier and let's people know the functionality exists.

Reply:
Oh... I see that -100 is used in the webModuleApplicationContext in PIH Core for pihcore.css... I don't remember if there was a reason we picked that... :)

Reply:
Thanks for the comments. Instead of adding empty css files, I checked if the file exists before adding the resource. 
Also removed empty js file.